src/imperative/cached_op.cc

```diff
@@ -32,247 +32,12 @@ DMLC_REGISTER_PARAMETER(CachedOpConfig);
 
 constexpr uint32_t kEidNotExist = std::numeric_limits<uint32_t>::max();
 
-const char CachedOp::FULL[] = "full";
```

Comment:
Could you describe the main changes made for cached_op.cc/h (code moved/removed/added)?

Reply:
added comments to cached_op.cc, moved the constant strings and the helper functions AddPrefix, CreateFullGraph, SetRefCounts, OptimizeRefCounts, CachedOpState to cached_op.h file. Refactored CreateForwardGraph and SetForwardRefCounts into its own function.

Reply:
Could you add these to PR description?